repositories/repositories.bzl

```diff
@@ -65,6 +65,33 @@ def ros2_repositories():
 
     ros2_repositories_impl()
 
+    maybe(
+        http_archive,
+        name = "attr",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:attr.BUILD.bzl",
+        sha256 = "25772f653ac5b2e3ceeb89df50e4688891e21f723c460636548971652af0a859",
+        url = "http://download.savannah.gnu.org/releases/attr/attr-2.4.47.src.tar.gz",
+        strip_prefix = "attr-2.4.47",
+    )
+
+    maybe(
+        http_archive,
+        name = "acl",
```

Comment:
See https://github.com/eclipse-iceoryx/iceoryx/blob/6c7c3a51491082a24f383aa4d9023f737be8269b/iceoryx_platform/qnx/include/iceoryx_platform/acl.hpp#L20, iceoryx always wants to use acl from the system (rootfs). So, I would remove builds for acl and attr and just put together which apt packages need to be installed.

Reply:
Looks like we only need `libacl1-dev`.